docs/site-example/site.conf

```diff
@@ -181,13 +181,18 @@
 
   -- Skip setup mode (config mode) on first boot
   -- setup_mode = {
-  --  skip = true,
+  --   skip = true,
```

Comment:
there really is a space missing, but that has nothing to do with this pull request